components/brave_news/browser/resources/Page.tsx

```diff
@@ -104,16 +87,16 @@ export default function FeedV2() {
   // Note: Whenever the feed is updated, if we're viewing the feed, scroll to
   // the top.
   React.useLayoutEffect(() => {
-    const root = document.querySelector<HTMLElement>('#root')
-    if (!root?.classList.contains(CLASSNAME_PAGE_STUCK)) {
-      return
+    if (ref.current) {
+      const rect = ref.current.getBoundingClientRect()
+      if (rect.top < window.innerHeight / 2) {
+        ref.current.scrollIntoView()
+      }
     }
-
-    ref.current?.scrollIntoView()
   }, [feedV2?.items])
 
   return <Root ref={ref as any} data-theme="dark">
-    <SidebarContainer>
+    <SidebarContainer className='brave-news-sidebar'>
```

Comment:
These CSS class names have been added so that the embedding UI can style the visibility and opacity of these elements itself (i.e. based on scroll position). The dependency on the existing NTP's `CLASSNAME_PAGE_STUCK` have been removed from this component.